cosmos/likecoin-public-testnet.json

```diff
@@ -0,0 +1,53 @@
+{
+  "chainId": "likecoin-public-testnet-5",
+  "chainName": "LikeCoin Testnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/likecoin-public-testnet/chain.png",
+  "rpc": "https://node.testnet-rpc.like.co/",
+  "rest": "https://node.testnet.like.co/",
+  "nodeProvider": {
+    "name": "like.co",
+    "email": "tech@like.co",
+    "website": "https://like.co"
+  },
+  "walletUrl": "https://likecoin-public-testnet-5.netlify.app/",
+  "walletUrlForStaking": "https://likecoin-public-testnet-5.netlify.app/",
```

Comment:
@williamchong Hello.
Could you remove `walletUrl` and `walletUrlForStaking`field?

Reply:
No problem, removed